File: mall-goods-api/src/main/java/life/eks/xmall/api/seller/SellerService.java
package life.eks.xmall.api.seller;

import life.eks.xmall.api.BaseService;
import life.eks.xmall.pojo.Seller;

/**
 * @author Eks OU
 */
public interface SellerService extends BaseService<Seller, String> {


    /**
     * 更改商家状态
     *
     * @param id     商家ids
     * @param status 要设置成的商家状态
     */
    void updateStatus(String id, String status);
}

File: mall-manager-web/src/main/resources/resources/js/xmall/service/sellerService.js
xmall.service('sellerService', function ($http) {

    this.list = function (pageNum, pageSize, entity) {
        return $http.post("../seller/list?pageNum=" + pageNum + "&pageSize=" + pageSize, entity);
    };

    this.findById = function (id) {
        return $http.get("../seller/findById?id=" + id);
    };

    this.updateStatus = function (sellerId, status) {
        return $http.post("../seller/updateStatus?id=" + sellerId + "&status=" + status);
    }
});
